### ISSUE TITLE
RangeError (index): Invalid value: Not in range 0..1023, inclusive: 1040

### ISSUE THREAD

**Issue by redsolver** [DavBfr/dart_pdf#21]
I got this error:
```
E/flutter ( 7635): [ERROR:flutter/shell/common/shell.cc(188)] Dart Error: Unhandled exception:
E/flutter ( 7635): RangeError (index): Invalid value: Not in range 0..1023, inclusive: 1040
E/flutter ( 7635): #0      List.[] (dart:core/runtime/libgrowable_array.dart:145:60)
E/flutter ( 7635): #1      PDFTTFFont.glyphAdvance (file:///home/redsolver/flutter/.pub-cache/hosted/pub.dartlang.org/pdf-1.0.7/lib/src/ttffont.dart:61:30)
E/flutter ( 7635): #2      new PDFTTFFont (file:///home/redsolver/flutter/.pub-cache/hosted/pub.dartlang.org/pdf-1.0.7/lib/src/ttffont.dart:45:19)
E/flutter ( 7635): #3      _PrintState._generateDocument (package:taggle/print.dart:175:27)
E/flutter ( 7635): <asynchronous suspension>
```

with this code
```
final pdf = new PDFDocument(
      deflate: zlib.encode,
    );
    var ttf = await rootBundle.load("assets/roboto.ttf");
    PDFTTFFont font = new PDFTTFFont(pdf, ttf); // Error here
```

**Comment by DavBfr** [comment]
Can you send me your roboto.ttf file?
When I download it from here: https://github.com/google/fonts/raw/master/apache/roboto/Roboto-Regular.ttf it works.
Thanks.

**Comment by redsolver** [comment]
It's Roboto Mono from https://fonts.google.com/specimen/Roboto+Mono

**Comment by DavBfr** [comment]
Fixed in pdf 1.0.8
